web/packages/selfhosted/test/polyfill/classic_frames_provided/test.ts

```diff
@@ -12,9 +12,28 @@ describe("Flash inside frame with provided ruffle", () => {
 
     it("polyfills inside a frame", async () => {
         await browser.switchToFrame(await browser.$("#test-frame"));
-        await browser.$("<ruffle-object />").waitForExist();
+        // TODO: After https://github.com/webdriverio/webdriverio/issues/13218 is fixed, use browser.$("<ruffle-object />")
+        await browser
+            .$(
+                () =>
+                    (
+                        document.querySelector(
+                            "#test-frame",
+                        ) as HTMLIFrameElement
+                    ).contentDocument?.body.querySelector(
+                        "<ruffle-object />",
```

Comment:
Obviously this will be removed when the underlying issue is fixed, but for now:

`querySelector` still can't take a tag name selector like this, that's just a JavaScript thing.

Reply:
Ah, good point, sorry. Will fix it up tomorrow - or feel free to take over now if you want to

Reply:
Fixed, thanks!